src/language-server/server.ts

```diff
@@ -35,9 +35,13 @@ const workspace = new GraphQLWorkspace(
   new LanguageServerLoadingHandler(connection),
   {
     clientIdentity: {
-      name: process.env["APOLLO_CLIENT_NAME"],
-      version: process.env["APOLLO_CLIENT_VERSION"],
-      referenceID: process.env["APOLLO_CLIENT_REFERENCE_ID"],
+      name: process.env["APOLLO_CLIENT_NAME"] || "Apollo Language Server",
+      version:
```

Comment:
Is there any significance to the `146d29c0-912c-46d3-b686-920e52586be6` fallback here?

Reply:
It has just moved around - see the old [src/language-server/engine/index.ts](https://github.com/apollographql/vscode-graphql/pull/163/files#diff-d675ce2b91731f4d5c3cdb539cea2811a6c1c5bfebcc59b64e56b16db8471333)